test_gpt2.cu

```diff
@@ -121,6 +121,7 @@ int main(int argc, char *argv[]) {
         if (argv[i][0] != '-') { exit(EXIT_FAILURE); } // must start with dash
         if (argv[i][1] == 'w') { model.use_master_weights = atoi(argv[i+1]); }
         else if (argv[i][1] == 'r') { model.recompute = atoi(argv[i+1]); }
+        else if (argv[i][1] == 'g' && argv[i][2] == 'e') { model.gelu_fusion = atoi(argv[i+1]); }
```

Comment:
mildly scary indexing into [2] here
 Should we add

```
if (!(strlen(argv[i]) == 2 || strlen(argv[i]) == 3)) { error_usage(); } // must be -x[y] (one dash, one or two letters)
```

like in train code?

Reply:
Good catch, hadn't noticed that was missing in test! Adding it now.

(it's a bit non-obvious that only works because strlen==2 means [2] is '\0', and that we'd need a better check if we wanted to use argv[i][3], but probably good enough for now)